test/jose-jws-test.html

```diff
@@ -53,18 +54,18 @@
                     });
 
                 });
-            }), true, "JWS message has been correctly verified");
+            }), true, "JWS message has been correctly verified 3");
```

Comment:
I think the '3' is a remnant of digging into the problem. Should be removed.

Reply:
and also fixed.